resources/js/Components/ItemIdSearchTextarea.vue

```diff
@@ -13,6 +13,9 @@
           :status="validationError ? validationError.type : 'default'"
           v-model="textareaInputValue"
       />
+      <template #label>
+			{{$i18n('item-form-id-input-label')}}
```

Comment:
Not a merge blocker, but the label automatically gets a font-weight of 700 (on Firefox for me) which appears bolder than the Figma illustration shows. I still think this can go ahead since we will be revamping the styles we missed once tokens are on development anyway